### DISCUSSION TITLE
macos Play Video Tips MovieFFMpeg: Failed to open codec 'aac_at'

### DISCUSSION THREAD

**Discussion by Jack7244** [AcademySoftwareFoundation/OpenRV#349] [General]
When compiling OpenRV on macOS and trying to play a video, it shows the error: ERROR: MovieFFMpeg: Failed to open codec 'aac_at'

**Comment by compaga** [comment]
Hi Jack7244, aac codec is not activated in Open RV by default. This codec is encumbered by a different license and royalties. You can customize the codec usage in https://github.com/AcademySoftwareFoundation/OpenRV/blob/main/cmake/dependencies/ffmpeg.cmake but it is your responsibility to validate your right to use the codec and/or require the proper license agreement.
